src/radio/dataflow.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Stubs to allow handlers to access smarthome.DataFlow
+ */
+export const DataFlowStub = {
```

Comment:
Will this just be for UDP or will the same file / `DataFlowStub` include other radios?

Reply:
This will include other radio stubs as well, one for each exported class in `smarthome.DataFlow` - `UdpRequestData`, `TcpRequestData`, and `HttpRequestData`